File: AlertBox1903F1/app/src/main/java/com/example/alertbox1903f1/MainActivity2.java
package com.example.alertbox1903f1;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

public class MainActivity2 extends AppCompatActivity {
ImageView img;
Button btn;
private int current_img_index;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);
        img=findViewById(R.id.imageView);
        btn=findViewById(R.id.button2);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                int[] images={R.mipmap.ic_img1,R.mipmap.ic_img2,R.mipmap.ic_img3,R.mipmap.ic_img4};
                current_img_index++;
                current_img_index=current_img_index % images.length;
                img.setImageResource(images[current_img_index]);
            }
        });
    }
}

File: FirstKotlinProject/app/src/main/java/com/example/firstkotlinproject/MainActivity2.kt
package com.example.firstkotlinproject

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.EditText
import android.widget.TextView
import java.util.*

class MainActivity2 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)
    }

    fun calculate(v:View){
var age=findViewById<EditText>(R.id.editTextTextPersonName)
var res=findViewById<TextView>(R.id.textView2)
        val current_year=Calendar.getInstance().get(Calendar.YEAR)
     res.text="Age :"+(current_year-age.text.toString().toInt())
    }
}